### PR TITLE
powerplatform-vscode Loc Checkin for WO 66644

### PR DESCRIPTION
**Pull request by v-hifuku-msft**
Localization completed in Work Order 66644

### CODE REVIEW

File: loc/translations-import/vscode-powerplatform.cs.xlf [hunk 1]
```diff
@@ -48,11 +48,9 @@
       </trans-unit>
       <trans-unit id="pacCLI.authPanel.welcome.whenInteractiveSupported">
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.cs.xlf:49]: Why were the translations for `pacCLI.authPanel.welcome.whenInteractiveSupported`, `pacCLI.authPanel.welcome.whenInteractiveNotSupported`, and `pacCLI.EnvOrSolutionTreeItem.toolTip` removed?

**Review by tehcrashxor — CHANGES_REQUESTED** [review]
After adding the `translations-import/vscode-powerplatform.*.xlf` files, you will also need to build the repo locally to generate the translated json files, and check those in.

See [Contributing.md](https://github.com/microsoft/powerplatform-vscode/blob/main/CONTRIBUTING.md) for general instructions on that, though the gist is

1. create an `msazure` org PAT [here](https://dev.azure.com/msazure/_usersSettings/tokens) with the Packaging -> Read scope
2. store that PAT in an environment variable `AZ_DevOps_Read_PAT`
3. run `npm install`
4. run `npm run ci`
5. check in the generated `*.json` translation files

File: loc/translations-import/vscode-powerplatform.cs.xlf (hunk 1 above)
Comment (v-hifuku-msft) [reply loc/translations-import/vscode-powerplatform.cs.xlf:49]: Brought back the missing translations.

**Comment by v-hifuku-msft** [comment]
@[tehcrashxor](https://github.com/tehcrashxor), I added the json files after fixing missing translations. Please review. Thanks a lot!

**Review by tehcrashxor — CHANGES_REQUESTED** [review]
@v-hifuku-msft , looks like checked in the `i18n/**/*.i18n.json` files generated by your local build, but did not check in the `package.nls.*.json` files that the build generates at the root of the repository.   Please commit those as well.

**Review by tehcrashxor — APPROVED** [review]
Tested the branch, and am successfully getting the updated strings in every part of the UI in the languages I tested.

**Pull request merged**